templates/lab-results/pdf_lab_results.go

```diff
@@ -0,0 +1,223 @@
+/*
+ * This example showcases the usage of creator templates by creating a sample
+ * lab results document.
+ *
+ * Run as: go run pdf_lab_results.go
+ */
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"log"
+	"os"
+	"text/template"
+
+	"github.com/unidoc/unipdf/v3/common"
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/core"
+	"github.com/unidoc/unipdf/v3/creator"
+)
+
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io.
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+
+	common.SetLogger(common.NewConsoleLogger(common.LogLevelDebug))
+}
+
+func main() {
+	c := creator.New()
+	c.SetPageMargins(15, 15, 180, 100)
+
+	// Read main content template.
+	mainTpl, err := readTemplate("templates/main.tpl")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Read lab results JSON data.
+	data, err := readResultData("lab_results.json")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Draw main content template.
+	tplOpts := &creator.TemplateOptions{
+		HelperFuncMap: template.FuncMap{
+			"extendDict": func(m map[string]interface{}, params ...interface{}) (map[string]interface{}, error) {
```

Comment:
Please remove the `extendDict` function as you don't seem to be using it.

Reply:
removed on latest commit